agent/util-scripts/pbench-tool-data-sink

```diff
@@ -221,21 +221,20 @@ class PromCollector(BaseCollector):
 
         self.logger.debug("PROM TERMINATED")
 
-        os.mkdir(self.benchmark_run_dir / f"tools-{self.tool_group}" / "prometheus")
-
         args = [
             "tar",
+            "--remove-files",
+            "--exclude",
+            "prometheus/prometheus_data.tar.gz",
             "-zcvf",
-            f"{self.benchmark_run_dir}/tools-{self.tool_group}/prometheus/prometheus_data.tar.gz",
+            f"{self.volume}/prometheus_data.tar.gz",
             "-C",
-            f"{self.benchmark_run_dir}/",
-            "prom_vol",
+            f"{self.volume}/../",
```

Comment:
This might be a bit easier to read if you had a `self.tool_group_dir` because seeing this construct makes me want to pop back up to find the original definition to see exactly what the parent directory is. Also, although it's a technicality in our case since it's unlikely we'd ever support Windows, it's cleaner to keep Python code free of OS dependencies, which is one reason why `Path() / 'directory'` is better than ` + '/directory'`